<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TableView</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>PlainTextEdit</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="PlaninTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionModelData"/>
   <addaction name="separator"/>
   <addaction name="actionAppend"/>
   <addaction name="actionInsert"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlignCenter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="actionFontBold"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/Open.png</normaloff>:/ico/ico/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/Save.png</normaloff>:/ico/ico/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAppend">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/Append.png</normaloff>:/ico/ico/Append.png</iconset>
   </property>
   <property name="text">
    <string>Append</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/Insert.png</normaloff>:/ico/ico/Insert.png</iconset>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/Delete.png</normaloff>:/ico/ico/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/Exit.png</normaloff>:/ico/ico/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionModelData">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/ModelData.png</normaloff>:/ico/ico/ModelData.png</iconset>
   </property>
   <property name="text">
    <string>ModelData</string>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/AlignLeft.png</normaloff>:/ico/ico/AlignLeft.png</iconset>
   </property>
   <property name="text">
    <string>AlignLeft</string>
   </property>
  </action>
  <action name="actionAlignCenter">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/AlignCenter.png</normaloff>:/ico/ico/AlignCenter.png</iconset>
   </property>
   <property name="text">
    <string>AlignCenter</string>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/AlignRight.png</normaloff>:/ico/ico/AlignRight.png</iconset>
   </property>
   <property name="text">
    <string>AlignRight</string>
   </property>
  </action>
  <action name="actionFontBold">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/ico/FontBold.png</normaloff>:/ico/ico/FontBold.png</iconset>
   </property>
   <property name="text">
    <string>FontBold</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
